package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "csmm",
-  "version": "1.19.0",
+  "version": "1.19.1",
```

Comment:
this get bumped on release, so lets leave it as 0 for now.
```suggestion
  "version": "1.19.0",
```

Reply:
sure